daisy_workflows/image_import/import_image.sh

```diff
@@ -57,22 +57,22 @@ SIZE_GB=$(awk "BEGIN {print int((${SIZE_BYTES}/1000000000)+ 1)}")
 
 echo "Import: Importing ${SOURCEPATH} of size ${SIZE_GB}GB to ${DISKNAME} in ${ZONE}." 2> /dev/null
 
-# Resize the disk if its bigger than 10GB and attach it.
+# Ensure the disk referenced by $DISKNAME is large enough to
+# hold the decompressed disk. For the common case, we initialize
```

Comment:
I'm not sure decompression is the correct term here, as that could indicate disk file is zipped. Maybe inflation? Disk could be in raw format where even inflation doesn't happen.

Reply:
Sure, 'inflated' sounds good to me.